packages/salesforcedx-vscode-lwc/test/vscode-integration/commands/forceLightningLwcStart.test.ts

```diff
@@ -227,17 +227,66 @@ describe('forceLightningLwcStart', () => {
         const executor = new ForceLightningLwcStartExecutor();
         const fakeExecution = new FakeExecution(executor.build());
         cliCommandExecutorStub.returns(fakeExecution);
+        devServiceStub.getBaseUrl.returns('http://localhost:3333');
 
         executor.execute({ type: 'CONTINUE', data: {} });
-        fakeExecution.stdoutSubject.next('Server up');
+        fakeExecution.stdoutSubject.next('Server up http://localhost:3333');
```

Comment:
let's use the actual output here `Server up on http://localhost:3333`

Reply:
@jag-sfdc small piece missing - this here and line 237 for 'Server up on'.

Reply:
The actual message that parses this now is in devServerService.test.ts, so added the actual output in that test. Will add in here as well for uniformity.